Cargo.toml

```diff
@@ -58,6 +60,11 @@ openssl = ["actix-http/openssl", "actix-tls/accept", "actix-tls/openssl"]
 # rustls
 rustls = ["actix-http/rustls", "actix-tls/accept", "actix-tls/rustls"]
 
+# Internal (PRIVATE!) features used to aid testing and cheking feature status.
+# Don't rely on these whatsoever. They may disappear at anytime.
+
```

Comment:
Please drop the empty line so it is clear it is related to the __compress flag.

Reply:
Done !